common/types/block.go

```diff
@@ -156,9 +158,14 @@ func ToBlockIDs(blocks []*Block) []BlockID {
 	return ids
 }
 
-// SortBlocks sort blocks by their IDs.
+// SortBlocks sort blocks tick height, if height is equal by lexicographic order.
 func SortBlocks(blks []*Block) []*Block {
-	sort.Slice(blks, func(i, j int) bool { return blks[i].ID().Compare(blks[j].ID()) })
+	sort.Slice(blks, func(i, j int) bool {
+		if blks[i].TickHeight < blks[j].TickHeight {
```

Comment:
i think it's worth adding a test here.

Reply:
i would rather make it private and move to mesh. it doesn't need to be exported and in common package if it used once

Reply:
yes. make sense